src/module.cpp

```diff
@@ -1114,10 +1114,12 @@ void Module::AddFunctionTemplateDeclaration(const TemplateParms *templateParmLis
 
     // TODO: inline / noinline check.
 
-    // TODO: vectorcall checks
-
     // ...
 
+    // We don't need to worry here about passing vectorcall/regcall specifier since we support these
```

Comment:
Doesn't this mean that we should check that these specifiers are not present and generate the error otherwise?

Reply:
This check is in parse.yy. I've added it to lit test as well.